stack-graphs/src/stitching.rs

```diff
@@ -485,54 +485,32 @@ impl ForwardPartialPathStitcher {
     /// paths that start with any of your requested starting nodes.
     ///
     /// Before calling [`process_next_phase`][] for the first time, you must ensure that `db`
-    /// contains all possible extensions of any of those initial partial paths.  You can retrieve a
-    /// list of those extensions via [`previous_phase_partial paths`][].
+    /// contains all of the possible partial paths that start with any of your requested starting
+    /// nodes.  You can retrieve a list of those extensions via [`previous_phase_partial paths`][].
     ///
     /// [`previous_phase_partial paths`]: #method.previous_phase_partial paths
     /// [`process_next_phase`]: #method.process_next_phase
     pub fn from_nodes<I>(
         graph: &StackGraph,
         partials: &mut PartialPaths,
-        db: &mut Database,
+        _db: &mut Database,
```

Comment:
Initially it was there for consistency with the `from_partial_paths` method, which _did_ use it. However, that was mostly because the constructors were already doing a bit of work. I've changed that at some point so that the constructor only sets up the stitcher, but doesn't do work until `process_next_phase` is called. Given that neither of the constructors use `db` anymore, I think we can remove it.